src/app/ClusterInfo.h

```diff
@@ -32,6 +32,25 @@ struct ClusterInfo
         kEventIdValid   = 0x03,
     };
 
+    bool IsAttributePathIncluded(const ClusterInfo & other) const
```

Comment:
Nit: consider some different name to help readers of code. I imagine some version of these could be easier to read:

```cpp
foo.IncludesAllAttributePathsFrom(bar);
foo.IsAttributePathSupersetOf(bar);
bar.IsAttributePathSubsetOf(foo);
```

Alternatively if we change the methods a bit, we could have code like:

```cpp
foo.GetAttributePath().Includes(bar.GetAttributePath())
```

however this may create too many separate types.

Reply:
agree, just update with IsAttributePathSupersetOf